qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -696,6 +696,9 @@ def __iter__(self):
     def current_idx(self):
         return self._history.currentItemIndex()
 
+    def current_item(self):
```

Comment:
if you are going to be making changes to the signature of this class you'll need to do the same to the corresponding webkit and abstract ones

Reply:
Fixed in c338feb5248adfa96c23eaef5de7a72bef985d41